src/common/FrameManager.ts

```diff
@@ -1144,57 +1129,49 @@ export class Frame {
   /**
    * Adds a `<script>` tag into the page with the desired url or content.
    *
-   * @param options - configure the script to add to the page.
-   *
+   * @param options - Options for the script.
    * @returns a promise that resolves to the added tag when the script's
    * `onload` event fires or when the script content was injected into the
    * frame.
    */
   async addScriptTag(
     options: FrameAddScriptTagOptions
   ): Promise<ElementHandle<HTMLScriptElement>> {
-    return this._mainWorld.addScriptTag(options);
+    return this.worlds[DEFAULT_WORLD].addScriptTag(options);
   }
 
   /**
    * Adds a `<link rel="stylesheet">` tag into the page with the desired url or
    * a `<style type="text/css">` tag with the content.
    *
-   * @param options - configure the CSS to add to the page.
-   *
+   * @param options - Options for the style link.
    * @returns a promise that resolves to the added tag when the stylesheets's
    * `onload` event fires or when the CSS content was injected into the
    * frame.
    */
   async addStyleTag(
     options: FrameAddStyleTagOptions
-  ): Promise<ElementHandle<Node>> {
-    return this._mainWorld.addStyleTag(options);
+  ): Promise<ElementHandle<HTMLStyleElement | HTMLLinkElement>> {
+    return this.worlds[DEFAULT_WORLD].addStyleTag(options);
```

Comment:
not sure if DEFAULT_WORLD is better than MAIN_WORLD? WDYT about calling it USER_WORLD or PAGE_WORLD to indicate that it is the where the user content runs? Otherwise, DEFAULT_WORLD sg